trieste/models/gpflow/models.py

```diff
@@ -586,17 +593,28 @@ class SparseVariational(
     GPflowPredictor,
     TrainableProbabilisticModel,
     SupportsCovarianceBetweenPoints,
+    SupportsGetInducingVariables,
```

Comment:
Also HasTrajectorySampler?

```suggestion
    SupportsGetInducingVariables,
    HasTrajectorySampler,
```

Reply:
Not sure why this didnt make Mypy upset